tests/release/pipelines/rhtap_service_push.go

```diff
@@ -0,0 +1,352 @@
+package pipelines
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+	"strings"
+
+	"github.com/devfile/library/v2/pkg/util"
+	ecp "github.com/enterprise-contract/enterprise-contract-controller/api/v1alpha1"
+	appservice "github.com/konflux-ci/application-api/api/v1alpha1"
+	appstudioApi "github.com/konflux-ci/application-api/api/v1alpha1"
+	"github.com/konflux-ci/e2e-tests/pkg/constants"
+	"github.com/konflux-ci/e2e-tests/pkg/framework"
+	"github.com/konflux-ci/e2e-tests/pkg/clients/github"
+	"github.com/konflux-ci/e2e-tests/pkg/utils"
+	"github.com/konflux-ci/e2e-tests/pkg/utils/tekton"
+	releasecommon "github.com/konflux-ci/e2e-tests/tests/release"
+	releaseapi "github.com/konflux-ci/release-service/api/v1alpha1"
+	tektonutils "github.com/konflux-ci/release-service/tekton/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"knative.dev/pkg/apis"
+)
+
+const (
+	rhtapServiceAccountName  = "release-service-account"
+	rhtapCatalogPathInRepo   = "pipelines/rhtap-service-push/rhtap-service-push.yaml"
+	rhtapGitSourceURL        = "https://github.com/redhat-appstudio-qe/devfile-sample-python-basic-test2"
+	rhtapGitSourceRepoName   = "devfile-sample-python-basic-test2"
+	rhtapGitSrcDefaultSHA    = "47fc22092005aabebce233a9b6eab994a8152bbd"
+	rhtapGitSrcDefaultBranch = "main"
+)
+
+var rhtapComponent *appservice.Component
+var gh *github.Github
+
+var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for rhtap-service-push pipeline", Label("release-pipelines", "rhtap-service-push"), func() {
+	defer GinkgoRecover()
+	var pyxisKeyDecoded, pyxisCertDecoded []byte
+
+	var devWorkspace = utils.GetEnv(constants.RELEASE_DEV_WORKSPACE_ENV, constants.DevReleaseTeam)
+	var managedWorkspace = utils.GetEnv(constants.RELEASE_MANAGED_WORKSPACE_ENV, constants.ManagedReleaseTeam)
+
+	var devNamespace = devWorkspace + "-tenant"
+	var managedNamespace = managedWorkspace + "-tenant"
+
+	var err error
+	var devFw *framework.Framework
+	var managedFw *framework.Framework
+	var rhtapApplicationName = "rhtap-app-" + util.GenerateRandomString(4)
+	var rhtapComponentName = "rhtap-comp-" + util.GenerateRandomString(4)
+	var rhtapReleasePlanName = "rhtap-rp-" + util.GenerateRandomString(4)
+	var rhtapReleasePlanAdmissionName = "rhtap-rpa-" + util.GenerateRandomString(4)
+	var rhtapEnterpriseContractPolicyName = "rhtap-policy-" + util.GenerateRandomString(4)
+	//Branch for creating pull request
+	var testPRBranchName = fmt.Sprintf("%s-%s", "pr-branch", util.GenerateRandomString(6))
```

Comment:
Can you change pr-branch to like `e2e-test` so if we were looking at the repo branches it would be more obvious it came from an e2e test run?

Reply:
Updated by adding "e2e-" prefix, thanks